doc/source/storage-pools.rst

```diff
@@ -0,0 +1,44 @@
+Storage Pools
+=============
+
+LXD supports creating and managing storage pools and storage volumes. General
+keys are top-level. Driver specific keys are namespaced by driver name. Volume
+keys apply to any volume created in the pool unless the value is overridden on
+a per-volume basis.
+
+`Storage Pool` objects are represent the dictionary and associated methods that
```

Comment:
`objects are represent the dictionary` - probably want to edit to `objects are represented by the dictionary`

Reply:
Good catch :)